pyaedt/edb_core/edb_data/sources.py

```diff
@@ -505,3 +505,20 @@ def pec_launch_width(self):
         """Get pec launch width."""
         txt = re.search(r"'PEC Launch Width'='.*?'", self._edb_properties).group()
         return txt.split("=")[1].replace("'", "")
+
+
+class ExcitationDifferential:
```

Comment:
Just missing the doctring. Maybe it is because this class will only be used internally.

Reply:
Perfect. Thanks